### PR TITLE
[WIP] Cleanup ordered consumer verticle

### PR DESCRIPTION
**Pull request by pierDipi**
## Proposed Changes

-
-
-

**Release Note**

```release-note

```

**Docs**

### CODE REVIEW

**Comment by pierDipi** [comment]
[artifacts-1425.tar.gz](https://github.com/knative-sandbox/eventing-kafka-broker/files/7496130/artifacts-1425.tar.gz)

File: data-plane/core/src/main/java/dev/knative/eventing/kafka/broker/core/OrderedAsyncExecutor.java [hunk 1]
```diff
@@ -30,10 +31,12 @@
   private final Queue<Supplier<Future<?>>> queue;
 
   private boolean isStopped;
+  private boolean inFlight;
 
   public OrderedAsyncExecutor() {
     this.queue = new ArrayDeque<>();
     this.isStopped = false;
+    inFlight = false;
```
Comment (matzew) [inline data-plane/core/src/main/java/dev/knative/eventing/kafka/broker/core/OrderedAsyncExecutor.java:39]: `this.` ?

**Comment by pierDipi** [comment]
Worked, retesting
/test upgrade-tests_eventing-kafka-broker_main

**Comment by pierDipi** [comment]
```
Error: init failed to get project from boskos: failed to get a "gke-project" from boskos: resources not found
```
/retest

**Pull request closed without merging**